src/input/code_templates/c/static_code/exi_bitstream_c.ctc

```diff
@@ -112,6 +112,42 @@ void exi_bitstream_init(exi_bitstream_t* stream, uint8_t* data, size_t data_size
 {%- endif %}
 }
 
+void exi_bitstream_reset(exi_bitstream_t* stream)
```

Comment:
Concerning a consistent api: the init function supports a `data_offset`, this should be then remembered, so that reset is able to do a consistent "reset".  Otherwise I would propose just to use init again instead of reset.

Reply:
Reset takes this case into account. The data offset from init is actually stored in _flag_byte_pos and then written back at reset.